testng.xml

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suite name="BLR_Analytics" verbose="10" parallel="tests">
 
-	<test name="Search in Jira" group-by-instances="true" verbose="3">
+	<test name="NewIssue" group-by-instances="true" verbose="3">
     		<classes>
-				<class name="tests.SearchJira"/>
```

Comment:
Return please class with test for Search page

Reply:
What class do you mean?  In master branch I see only 
<test name="Search in Jira" group-by-instances="true" verbose="3">
    		<classes>
				<class name="tests.SearchJira"/>
		    </classes>
</test>
These lines are present in testng.xml. They were moved from 6 to 10 line